src/core/math/vec3/lerp.js

```diff
@@ -5,7 +5,7 @@ const create = require('./create')
  * Performs a linear interpolation between two vec3's
  *
  * @param {vec3} out the receiving vector
- * @param {Number} t interpolation amount between the two inputs
+ * @param {Number} t interpolant (0.0 to 1.0) applied between the two inputs
```

Comment:
are you sure about this ? I do not get the word 'interpolant' , interpolation amount seems clearer
but good addition of the 0.0 to 1.0

Reply:
I’m not a mathematician, but that’s the name given by the website. 😃